### ISSUE TITLE
Figure Number Issue in 14-1.md 

### ISSUE THREAD

**Issue by marofmar** [Atcold/pytorch-Deep-Learning#461]
Hi, there is inconsistency in the name of the figure and how it is indicated in the following explanation. 

For example, in figure 5 case, it is actively used to describe the content but it is called figure 19 in the explanation. As the figure itself contains the number, it makes sense, but this hinders a smooth understanding of the lecture note. 

I think there are two solutions to this: 

1. Remove figure name part in the figure picture itself or
2. Change figure name in the explanation 

```

<center>
<img src="{{site.baseurl}}/images/week14/14-1/Fig5.png" style="background-color:#DCDCDC;" /><br>
Figure 5.
</center>

A simple example of a factor graph is shown in Figure 19 (top). The energy function is the sum of four factors:






<center>
<img src="{{site.baseurl}}/images/week14/14-1/Fig6.png" style="background-color:#DCDCDC;" /><br>
Figure 6.
</center>

The factor graph shown in Figure 20 is a log domain factor graph for linear structured models ("simple energy-based factor graphs" we are talking about)

**Comment by marcozullich** [comment]
I would also add that figures might need captioning as in all of the other documents.
Would you be so kind as to add captioning to the EN version of 14-1?

**Comment by Atcold** [comment]
@marofmar, thanks for pointing this out.
Can you convert the bitmap caption in textual caption? In this way we can translate it to other languages as well.
Moreover, could you add captions to all the other figures as well?

Thanks!

**Comment by marcozullich** [comment]
Btw, since I hadn't got an answer to my first reply, I've already started putting captions while I'm translating the Italian version, in case @marofmar won't answer.

**Comment by marofmar** [comment]
Hi @marcozullich, sorry that I checked your reply so late,
@marcozullich @Atcold noted!! I will add captions to all the other figures in [EN]14-1.md

**Comment by marofmar** [comment]
@Atcold bitmap caption which is included in the bitmap image is already used in the textual caption to explain the content. I modified a bit for consistency, but it results in messy ordering in the numbering system in figures of 14-1.md. 

I think modifying the bitmap image itself, cropping it out the bitmap caption in the image, would be a better option. Anyhow please check that out on the github.io webpage and the PR of mine.

**Comment by Atcold** [comment]
Yes, we should crop the images, put the caption in written form, and update the references to the figure number to reflect the correct numbering.

**Comment by marofmar** [comment]
As the PR of mine on Oct 17 has not been merged yet, and won't be, I think cropping the images would be the only work that left! @marcozullich @Atcold

**Comment by marofmar** [comment]
oh I just checked the note again, and will make a quick fix and PR again, thanks @marcozullich !

**Comment by Atcold** [comment]
Thanks! Getting to it right now!

**Comment by marcozullich** [comment]
I see now that the fig. numbers should be fine, is it @marofmar ?
Now I guess we'd have to put some captions on the images
